src/plugin.js

```diff
@@ -4,12 +4,10 @@
  */
 
 /* eslint-env browser */
-
-import Vue, { version as vueVersion } from 'vue';
+import * as Vue from 'vue';
```

Comment:
If someone uses this component loaded as ES6 module with Vue v2.x, the incompatibility warning will not be displayed anymore:

**Expected:**

<img src="https://user-images.githubusercontent.com/56868128/180757104-696db639-7882-44bd-9591-d46cf1090665.png" width="800px">

**Actual:**

<img src="https://user-images.githubusercontent.com/56868128/180757129-2000d6a5-fc45-477c-9aa5-ed42db12c48f.png" width="600px">

The `Vue.version` is `undefined` here. For Vue v2.x, the `Vue.default.version` contains the version.

Reply:
Addressed in 66f1191.